src/components/Explore/TokenRow.tsx

```diff
@@ -462,35 +462,32 @@ export default function LoadedRow({
   const tokenSymbol = token?.symbol
   const tokenData = data[tokenAddress]
   const [favoriteTokens, updateFavoriteTokens] = useAtom(favoritesAtom)
+  const theme = useTheme()
   // TODO: remove magic number colors
   const tokenPercentChangeInfo = (
     <>
       {tokenData.delta}%
       <ArrowCell>
         {Math.sign(tokenData.delta) > 0 ? (
-          <ArrowUpRight size={16} color={'#57bd0f'} />
+          <ArrowUpRight size={16} color={theme.green1} />
         ) : (
-          <ArrowDownRight size={16} color={'red'} />
+          <ArrowDownRight size={16} color={theme.red1} />
         )}
       </ArrowCell>
     </>
   )
-  const theme = useTheme()
 
   /* handle favorite token logic */
-  const isTokenFavorited = atom<boolean>(favoriteTokens.includes(tokenAddress))
-  const [tokenFavorited, setTokenFavorite] = useAtom(isTokenFavorited)
   const toggleFavoriteToken = () => {
-    let updatedFavoriteTokens = favoriteTokens
-    if (tokenFavorited) {
+    let updatedFavoriteTokens = favoriteTokens.slice()
+    if (favoriteTokens.includes(tokenAddress)) {
       updatedFavoriteTokens = favoriteTokens.filter((address: string) => {
         return address !== tokenAddress
       })
     } else {
       updatedFavoriteTokens.push(tokenAddress)
```

Comment:
then make this `updatedFavoriteTokens = [...favoriteTokens, tokenAddress]`

Reply:
ok thank you thank you